mdoc/Mono.Documentation/MDocUpdater.cs

```diff
@@ -1633,7 +1633,15 @@ public void DoUpdateType2 (string message, XmlDocument basefile, TypeDefinition
                         oldmember.SetAttribute ("MemberName", memberName);
                     }
 
-                    AddEiiNameAsAttribute(info.Member, oldmember, memberName);
+                    AddEiiNameAsAttribute(info.Member, oldmember, memberName);
+
+                    // Check valid AssemblyInfo and delete
+                    if (typeEntry.Framework.IsFirstFrameworkForType(typeEntry))

```

Comment:
So this check here is good ... this will take care of this logic at the member level. However, we still have to check it at the type level :) 

You can probably add that in the UpdateType function, before mdoc adds assemblies, here
https://github.com/mono/api-doc-tools/blob/483c3fca179bbc4ade06417bf2055c0f953c89d3/mdoc/Mono.Documentation/MDocUpdater.cs#L2243

Reply:
This right here ... this should be added back :)